app/javascript/packages/webauthn/webauthn-input-element.spec.ts

```diff
@@ -1,13 +1,19 @@
 import sinon from 'sinon';
 import quibble from 'quibble';
 import type { IsWebauthnPasskeySupported } from './is-webauthn-passkey-supported';
+import type { isWebauthnPlatformAvailable } from './is-webauthn-platform-authenticator-available';
 
 describe('WebauthnInputElement', () => {
   const isWebauthnPasskeySupported = sinon.stub<
     Parameters<IsWebauthnPasskeySupported>,
     ReturnType<IsWebauthnPasskeySupported>
   >();
 
+  const isWebauthnPlatformAvailable = sinon.stub<
+    Parameters<isWebauthnPlatformAvailable>,
+    ReturnType<isWebauthnPlatformAvailable>
```

Comment:
Ah, re: my previous comment, I forgot we reference the type here. In that case, maybe it's just enough to fix up the capitalization, like what we have with the first stub.